### ISSUE TITLE
char field unicode problem on Mac OSX 10.5

### ISSUE THREAD

**Issue by GoogleCodeExporter** [maximemaher/pyodbc#27]
```
> What version of the product are you using? On what operating system?
Mac OSX 10.5, Python 2.5, latest stable pyodbc, MS SQL DB using the
Actual SQL ODBC driver.

I am getting the following error message when trying to access certain
char fields:

Assertion failed: (size >= 0), function PyUnicodeUCS2_EncodeUTF8, file
Objects/unicodeobject.c, line 1416. 
```

Original issue reported on code.google.com by `squishywaffle@gmail.com` on 20 Jan 2009 at 9:17

**Comment by GoogleCodeExporter** [comment]
```
I believe this is fixed in 2.1.8, though I was not able to reproduce it 
exactly.  I'm going to close this, but please reopen if it is not fixed!  
Thanks.
```

Original comment by `mkleehammer` on 6 Sep 2010 at 6:10
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `mkleehammer` on 21 Nov 2010 at 4:44
- Changed state: **Complete**
